Say/ViewController.swift

```diff
@@ -54,6 +54,7 @@ class ViewController: NSViewController {
     /// Combo box for voices. Default is decided by system locale
     @IBOutlet var voiceComboBox: NSComboBox! = nil;
     /// Save panel for "Export" menu
+    @IBOutlet var URLFeild: NSTextField! = nil;
```

Comment:
The semicolon is redundant. (It is fault of original code.)
Will you remove all the semicolons of the IBOutlets?